components/identities/actions/changeAddressModal.tsx

```diff
@@ -4,24 +4,25 @@ import React, { FunctionComponent, useEffect, useState } from "react";
 import { isHexString, minifyAddress } from "../../../utils/stringService";
 import styles from "../../../styles/components/modalMessage.module.css";
 import Button from "../../UI/button";
-import { hexToDecimal } from "../../../utils/feltService";
+import { hexToDecimal, stringToFelt } from "../../../utils/feltService";
 import ConfirmationTx from "../../UI/confirmationTx";
 import { useNotificationManager } from "../../../hooks/useNotificationManager";
 import { NotificationType, TransactionType } from "../../../utils/constants";
+import { Identity } from "../../../utils/apiObjects";
 
 type ChangeAddressModalProps = {
   handleClose: () => void;
   isModalOpen: boolean;
   callDataEncodedDomain: (number | string)[];
-  domain?: string;
+  identity?: Identity;
```

Comment:
No I don't mean that. 

Just mean that it should be mandatory in theory (you can't open this modal without any identity), `domain` wasn't mandatory before because it made the app crash sometimes on `Property of undefined` error.